dwml-data-parsers/parameter.js

```diff
@@ -72,9 +72,23 @@ var parameterParser = {
 
     while (i < values.length) {
       var currentValue = values[i];
+
+      currentTimeFrame = timeFrames[timeFrameCounter];
       if (currentValue.name === 'value') {
         currentTimeFrame = timeFrames[timeFrameCounter];
-        results.push(_.extend({}, currentTimeFrame, {value: currentValue.content }));
+        results.push(_.extend({}, currentTimeFrame, {value: currentValue.content }));  
+        timeFrameCounter++;
+      }
+      if (currentValue.name === 'weather-conditions') {
```

Comment:
I can't do all the checks in that if statement because summary will always be present, and we need to return that. I agree with making a hasChildren though, to clean up the code. Give me a second